src/diffusers/utils/import_utils.py

```diff
@@ -570,6 +570,8 @@ class DummyObject(type):
     `requires_backend` each time a user tries to access any method of that class.
     """
 
+    _is_onnx = False
```

Comment:
This flag doesn't need to be hard coded. It won't be present when trying to create `OnnxPipelines` without the `onnxruntime` installed. So an error message will be thrown.